ImageNet/train/google_amoeba_net_d_tpu_tensorflow17.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "v1.0",
+  "author": "Google Cloud TPU",
+  "authorEmail": "cloud-tpu-dawnbench@googlegroups.com",
+  "framework": "TensorFlow v1.7rc1",
+  "codeURL": "https://github.com/tensorflow/tpu/tree/master/models/experimental/amoeba_net",
```

Comment:
Can you either change this to a specific commit hash or include the commit hash in `misc`?

Reply:
Done